src/ui_manager.py

```diff
@@ -144,16 +145,16 @@ def start_game(self) -> bool:
             # roi = Region of interest. It reduces the search area and can be adapted within game.ini
             # note that any ui_rois are in 1080p coordinates (and will automatically be converted if using 720p)
             # by running >> python src/screen.py you can visualize all of the currently set region of interests
-            found_btn, btn_pos = self._template_finder.search("PLAY_BTN", img, roi=self._config.ui_roi["go_btn"], threshold=0.8)
+            found_btn = self._template_finder.search("PLAY_BTN", img, roi=self._config.ui_roi["go_btn"], threshold=0.8)
             score_enabled = self._template_finder.last_score
             # same as above, just with different template
             self._template_finder.search("PLAY_BTN_GRAY", img, roi=self._config.ui_roi["go_btn"], threshold=0.8)
             score_disabled = self._template_finder.last_score
             # found_btn is a bool (True or False), if a play btn was found and it is not grayed out, then we can proceed
-            found_btn = found_btn and score_enabled > score_disabled
-            if found_btn:
+            play_active = found_btn and score_enabled > score_disabled
```

Comment:
if for whatever reason play_btn_gray is not found the score will be the same. Can you reset the "last_score" in template finder search function()? Just add self.last_score = -1 whenever we call search()

Reply:
I don't think that has to do with anything I've modified here, right? If I remember right I thought I just consolidated a couple lines. score_disabled will be the score of the last attempt even if not found, if its not found it'll just show the horrible score. Seems to behave fine, it waits appropriately for the play button to activate.

could probably rework this whole section though using multiSearch and passing ["PLAY_BTN_GRAY","PLAY_BTN"] with best_match=True if desired.

Reply:
No, nothing you introduced, just thought that would be nice  fix while we are at it : )

Reply:
I ended up reworking this, have only tested one time. Hope it works :p